tools/download-deps.sh

```diff
@@ -116,4 +116,7 @@ do_curl_binary "stoml" "${STOML_URL}" "${BIN_DIR}"
 tools=$("${BIN_DIR}"/stoml "${DEP_FILE}" .)
 for tool in $tools; do
     download_dependency "${tool}" "${BIN_DIR}"
-done
\ No newline at end of file
+done
+
+echo "Installing golangci-lint"
+curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.42.1
```

Comment:
Does adding `golangci` to the `tools` package not auto-magically install the binary for you?

Reply:
The documentation recommended not to install it via `go get` but through a binary.  This seemed like the safer option, but I can add it as an import.